log-classifier.cabal

```diff
@@ -46,6 +47,7 @@ library
                      , optparse-applicative
                      , reflection
                      , regex-tdfa
+                     , safe-exceptions
```

Comment:
Are you sure we need this? http://hackage.haskell.org/package/universum-1.2.0/docs/Universum-Exception.html

Reply:
I've tried importing `tryDeep` from `Universum.Exceptions` but didn't work. I'm not sure what they mean by `re-export`.
`cardano-sl` 's `new-wallet` component is importing `safe-exceptions` even though it's using `universum`
https://github.com/input-output-hk/cardano-sl/blob/develop/wallet-new/cardano-sl-wallet-new.cabal#L175

Reply:
Ok, keep it there, just try using functions from `Universum` as much as possible.